src/Reactduino.h

```diff
@@ -22,9 +22,15 @@ typedef int32_t reaction;
 #define REACTION_TYPE_STREAM 2
 #define REACTION_TYPE_INTERRUPT 3
 #define REACTION_TYPE_TICK 4
+#define REACTION_TYPE_INPUT_CHANGE 5
 
 #define REACTION_TYPE(x) ((x) & REACTION_TYPE_MASK)
 
+#define INPUT_STATE_HIGH    HIGH
+#define INPUT_STATE_LOW     LOW
+#define INPUT_STATE_ANY     0xff
+#define INPUT_STATE_UNSET   0xfe
```

Comment:
This is a bit of a nitpick - but could I get these in capitals?

Not very important but I usually use hex letters in caps and it's just a bit more consistent  with some of the other defs.